features/providers/check_nonpassport_autogrant.feature

```diff
@@ -0,0 +1,163 @@
+    Then I click 'Save and continue'
+    Then I am on the "may need to pay towards legal aid" page
+    Then I click 'Save and continue'
+    Then I should be on a page showing 'Provide details of the case'
+    When I click link 'Latest incident details'
+    Then I should be on a page showing 'Latest incident details'
+    Then I fill "application_merits_task_incident_told_on_3i" with "5"
+    Then I fill "application_merits_task_incident_told_on_2i" with "5"
+    Then I fill "application_merits_task_incident_told_on_1i" with "20"
+    Then I fill "application_merits_task_incident_occurred_on_3i" with "4"
+    Then I fill "application_merits_task_incident_occurred_on_2i" with "4"
+    Then I fill "application_merits_task_incident_occurred_on_1i" with "20"
+    Then I click 'Save and continue'
+    Then I should be on a page showing 'Opponent details'
+    Then I fill "application-merits-task-opponent-full-name-field" with "Bob"
+    Then I choose "application-merits-task-opponent-understands-terms-of-court-order-true-field"
+    Then I choose "application-merits-task-opponent-warning-letter-sent-true-field"
+    Then I choose "application-merits-task-opponent-police-notified-true-field"
+    Then I fill "application-merits-task-opponent-police-notified-details-true-field" with "Mike non passported test"
+    Then I choose "application-merits-task-opponent-bail-conditions-set-field"
+    Then I click 'Save and continue'
+    Then I should be on a page showing "Provide a statement of case"
+    Then I fill "application-merits-task-statement-of-case-statement-field" with "Mike non passported SOC"
+    Then I click 'Save and continue'
+    Then I should be on a page showing 'Provide details of the case'
+    When I click the first link 'Chances of success'
+    Then I should be on a page showing "Is the chance of a successful outcome 50% or better?"
+    Then I choose "proceeding-merits-task-chances-of-success-success-likely-true-field"
+    Then I click 'Save and continue'
+    Then I should be on a page showing 'Provide details of the case'
+    When I click the last link 'Chances of success'
+    Then I should be on the 'chances_of_success' page showing 'Is the chance of a successful outcome 50% or better?'
+    When I choose 'Yes'
+    And I click 'Save and continue'
+    Then I click 'Continue'
+    Then I should be on the 'gateway_evidence' page showing 'Upload supporting evidence'
+    And I click 'Save and continue'
+    Then I should be on a page showing "Check your answers and submit application"
+    Then I click 'Submit and continue'
+    Then I should be on a page showing "Application complete"
+    Then I click 'View completed application'
+    Then the application must be manually reviewed in CCMS
+
+@javascript @vcr
+  Scenario: I am able to create a non passported application without Income or Cap Contribs and with no restrictions
+    Given the setting to manually review all cases is enabled
```

Comment:
This might be fine, but I'm not clear why this test requires the setting to be enabled when the one above doesn't.

If the test here is to check that a non-passported application with no contribs/restrictions doesn't auto-grant, isn't the outcome of that being affected by manually changing the setting?

Or is it that a non-passported application with no contribs/restrictions _does_ auto-grant and we're testing that we can force it to auto-grant by changing. the setting? In which case I think the scenario might just need rewording to make that clear.